src/d3-hwschematic.js

```diff
@@ -189,6 +189,8 @@ export default class HwSchematic {
     _applyLayout() {
         var root = this.root;
 
+        this._applyLayoutLinks();
```

Comment:
Why was the move required?
(It is not a problem, I am asking from a curiosity.)

Reply:
Hi. Moving applyLayoutLinks means that the links are drawn before (ie underneath) the nodes. Drawing the node over the top of the link hides the small excursion of the link into the node (the output link in the top picture extends slightly into the node). It's a bit hacky - but probably better than trying to alter the start/end coordinates for each link to match a specific node type.

Reply:
PS: my understanding is that D3/SVG has no capability to specifically assign a z-order independent of creation/append order.